zgl.zig

```diff
@@ -428,7 +428,7 @@ pub fn vertexAttribPointer(attribindex: u32, size: u32, attribute_type: Type, no
         @enumToInt(attribute_type),
         b2gl(normalized),
         @intCast(c.GLint, stride),
-        if (relativeoffset != null) @intToPtr(*c_void, relativeoffset.?) else null,
+        if (relativeoffset != null) @intToPtr(?*c_void, relativeoffset.?) else null,
```

Comment:
this one was weird, I would have assumed that the "if" will take care of the case where relativeoffset is 0, but for some reason the cast was still beeing triggered when running a testprogram ... (maybe some problem with the zig version I used?)

Reply:
this might have to do with the type of the expression itself, but i'm not sure. i think it's correct this way